### PR TITLE
Add pay button animations/messages

### PR DESCRIPTION
**Pull request by stephen-gilbert-cko**

### CODE REVIEW

**Comment by ioan-ghisoi-cko** [comment]
@stephen-gilbert-cko it's 2020. No JQuery, please.

File: src/front-end/index.html [hunk 1]
```diff
@@ -65,19 +67,28 @@
...
     <div id="button-dots" class="dots white-dots">
       <span></span>
       <span></span>
       <span></span>
     </div>
+
+    <div class="checkmark draw"></div>
+
+    <!-- Cross -->
+    <svg class="container container--fail" viewBox="0 0 50 50">
```
Comment (ioan-ghisoi-cko) [inline src/front-end/index.html:81]: Does the SVG have to be referenced like this or can it be a separate file with the SVG?

File: src/front-end/index.html [hunk 2]
```diff
@@ -65,19 +67,28 @@
...
+      <path class="container__path cross" fill="none" d="M16 16 34 34 M34 16 16 34"></path>
+    </svg>
+    <script>$(".container").toggle();</script>
```
Comment (ioan-ghisoi-cko) [inline src/front-end/index.html:84]: don't write js here, write it in the js file

**Comment by stephen-gilbert-cko** [comment]
@ioan-ghisoi-cko ok 😳 will convert to standard js

File: src/front-end/index.html (hunk 1 above)
Comment (stephen-gilbert-cko) [reply src/front-end/index.html:81]: I could put it in a separate file and embed in the HTML within an object, I'd then need to move the CSS to that SVG file also. Not sure how I would reference it in my javascript to change display properties though...

File: src/front-end/index.html (hunk 2 above)
Comment (stephen-gilbert-cko) [reply src/front-end/index.html:84]: Moved this in the latest change

**Comment by stephen-gilbert-cko** [comment]
@ioan-ghisoi-cko Have updated the code now. Let me know whether you're happy with the SVG or if you want me to have another look at that.

**Pull request merged**